PYIC-7470: Add bulk migration batchIds to tactical store VCs

Wynndow opened the pull request:
**Not to be merged before https://github.com/govuk-one-login/ipv-core-common-infra/pull/1064**
<!-- Provide a general summary of your changes in the Title above -->
<!-- Include the Jira ticket number in square brackets as prefix, eg `[P4-XXXX] PR Title` -->

## Proposed changes

### What changed

Add bulk migration batchIds to tactical store VCs

### Why did it change

This adds the batch ID from the bulk migration to the VCs in the tactical store. This allows for us to more easily deal with having to roll back a batch.

The Bravo team will be responsible for deleting VCs from EVCS. We can then run the script again and remigrate the VCs whos batch IDs match a value in config.

And for on-demand migrations in the check-existing-identity lambda we can follow similar logic - check if the users tactical VCs have a matching batch ID, and remigrate if they do.

### Issue tracking
<!-- List any related Jira tickets or GitHub issues -->
<!-- List any related ADRs or RFCs -->
<!-- Delete/copy as appropriate -->

- [PYIC-7470](https://govukverify.atlassian.net/browse/PYIC-7470)


[PYIC-7470]: https://govukverify.atlassian.net/browse/PYIC-7470?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Wynndow committed:
PYIC-7470: Add bulk migration batchIds to tactical store VCs

This adds the batch ID from the bulk migration to the VCs in the
tactical store. This allows for us to more easily deal with having to
roll back a batch.

The Bravo team will be responsible for deleting VCs from EVCS. We can
then run the script again and remigrate the VCs whos batch IDs match a
value in config.

And for on-demand migrations in the check-existing-identity lambda we
can follow similar logic - check if the users tactical VCs have a
matching batch ID, and remigrate if they do.

Joe-Edwards-GDS reviewed (commented):
This looks ok - hopefully the extra checks for mismatched batches etc. won't be necessary, but shouldn't hurt.

However, I think we'll need to also update the on-demand migration logic to clear the batch ID (in `EvcsMigrationService`) - so that running an on-demand migration will correctly supersede the rolled back migration.

(If this happens before EVCS have reverted on their side we may see errors, but that's probably an acceptable impact)

Wynndow committed:
PYIC-7470: Remove extra call to VC service

We're already updating the VCs in the tactical store in the migration
service, so we just need to make sure we're setting the batchId to null.
